src/object/block.hpp

```diff
@@ -62,6 +63,9 @@ class Block : public MovingObject
   float m_bounce_offset;
   float m_original_y;
 
+private:
+  Flip m_flip;
+
```

Comment:
~~Same comment as my second review (I think) - would it be possible to place this variable in `MovingSprite` instead?~~

EDIT: I just saw it inherits from `MovingObject` and not `MovingSprite` - perhaps that should be changed? (If it makes sense and Block is consistent with MovingSprite)

Reply:
I honestly don't feel _at home_ enough with the code yet to evaluate and change that. It seems it might be better to make `Block` inherit from `MovingSprite`, but I don't think I'm the right person for the job. Either way, that's not really related to flipping and should probably be done in a separate PR at some point.